README.md

```diff
@@ -29,6 +29,8 @@ To compile and installing to in your local Maven repository:
     mvnw clean install
     mvnw package
 
+If you have deb maven installed, you can use the `mvn` command instead.
```

Comment:
Add instructions for people who have maven installed but might not know its CLI command